utils/report.c

```diff
@@ -761,6 +764,14 @@ void report_sort_tasks(struct uftrace_data *handle, struct rb_root *name_root,
 	}                                                                                          \
 	FIELD_STRUCT(_id, _name, _func, _header, 10)
 
+#define FIELD_FUNC_SIZE(_id, _name, _field, _func, _header)                                        \
+	static void print_##_func(struct field_data *fd)                                           \
+	{                                                                                          \
+		struct uftrace_report_node *node = fd->arg;                                        \
+		pr_out("%10" PRIu64 "", node->_field);                                             \
+	}                                                                                          \
+	FIELD_STRUCT(_id, _name, _func, _header, 10)
+
```

Comment:
Can we reuse the existing `FIELD_CALL`?  Maybe you want to rename it to `FIELD_UINT` or something.

Reply:
Ok, I will try it. Thanks for advice!

Reply:
Fixed commit is updated. Please check it.